modules/light-clients/08-wasm/types/msgs.go

```diff
@@ -37,6 +39,30 @@ func (m MsgStoreCode) ValidateBasic() error {
 	return nil
 }
 
+// NewMsgRemoveCodeHash creates a new MsgRemoveCodeHash instance
+//
+//nolint:interfacer
+func NewMsgRemoveCodeHash(signer string, codeHash []byte) *MsgRemoveCodeHash {
+	return &MsgRemoveCodeHash{
+		Signer:   signer,
+		CodeHash: codeHash,
+	}
+}
+
+// ValidateBasic implements sdk.Msg
```

Comment:
I guess it doesn't hurt to tweak this to `HasValidateBasic`.

Reply:
done